release_notes/v10_0_0.html.md

```diff
@@ -0,0 +1,63 @@
+# v10.0.0 (2019-12-03)
+### Bug Fixes
+* Fixes an issue that prevented the experimental `Date` component being controlled using the value prop (Component: Date)
+* Fixes issue of component not having proper data attributes needed for testing. (Components: ActionPopover)
+* Fixes: Link Component passes non-html props to the DOM (Components: Link)
+* Fixes incorrect validation icon size' (Components: Date Range/TextBox/Date Input/Form/Radio Button/Select/Textarea)
+* Fixes issue with menu button event propagation. (Components: ActionPopover)
+* Moves validation concatenation out of render method and into state and handles on update as well. (Component: Date)
+* Fixes: Label Help too low on deprecated Textarea. (Components: Deprecated Textarea)
+* Ensure Pager uses I18n for navigation text.
+* Fixes: OnBlur not working properly on IE11 when multiple selection enabled. (Component: Select)
+* Fixed issue where component does not support using it as uncontrolled (Component: SimpleColorPicker)
+* Fixes an issue with wrong mouse cursor when hovering or dragging items (Component: Date)
+* Fixes: Hover state activated on button when help icon is hovered. (Component: Label)
+* Fixes wrong icon color display when buttonType prop is not provided (Component: Button).
+* Fixes unnecessary onBlur triggering while removing selected items in multi-select mode. (Components: Select)
+* Fixes issue of incorrectly working pagination in case of only one page being available. (Components: Table)
+* Fixes incorrect Help Icon colour in the Classic story. (Components: Help)
+* Fixes association between Label and input components by correctly passing id prop to the Label htmlFor prop (Component: FormField)
+* Fixes wrong input icon position on demo site (component: TextBox)
+* button type set to submit. Enbales save button to still submit. (component: FormButton)
+* Fixes console warnings displayed when mounting Switch Component. (Components: Switch)
+* Implements missing controlled component functionality. (Component: ButtonToggleGroup)
+* Fixes erroneous appearance of Unknown component proptypes in story. (component: Switch)
+* Fixes back arrow and label misalignment in Classic story. (Components: Pages)
+* Adds fix to prevent picker opening when switching browser tabs and to prevetn an extra onBlur event when input icon clicked (Component: Date)
```

Comment:
```suggestion
* Adds fix to prevent picker opening when switching browser tabs and to prevent an extra onBlur event when input icon clicked (Component: Date)
```